src/coreclr/nativeaot/Bootstrap/main.cpp

```diff
@@ -222,7 +223,12 @@ int main(int argc, char* argv[])
     if (initval != 0)
         return initval;
 
+#if !defined(DEBUG) || !defined(_WIN32)
```

Comment:
```suggestion
#if defined(DEBUG) && defined(_WIN32)
    // quick_exit works around Debug UCRT shutdown issues: https://github.com/dotnet/runtime/issues/108640
    quick_exit(__managed__Main(argc, argv));
#else
```
Nit: Positive conditions are easier to understand. Also, matches the polarity of ifdef in the other place we are applying the workaround.